lsp-mode.el

```diff
@@ -4958,8 +4961,11 @@ RENDER-ALL - nil if only the signature should be rendered."
                (insert str)
                (visual-line-mode 1)
                (current-buffer))
+             lsp-signature-posframe-params
              :position (point)
-             lsp-signature-posframe-params)
+             :background-color (face-attribute 'lsp-signature-posframe :background)
+             :foreground-color (face-attribute 'lsp-signature-posframe :foreground)
+             :border-color (face-attribute 'lsp-signature-posframe :background))
```

Comment:
I havent tested it but it seems to me that it should be: `(append lsp-signature-posframe-params (list :position ...))`. The last argument of `apply` should be list.

Reply:
fixed